prework/week2LoopsExercise.js

```diff
@@ -39,10 +39,10 @@
 // run without ever stopping). Then fix the loop so that it stops when
 // counterFour is equal to -100.
 
-let counterFour = 1;
+const counterFour = 1;
 while (counterFour < 2) {
-    console.log('HELP ME!')
-    counterFour--;
+  console.log('HELP ME!')
+  counterFour--;
```

Comment:
this needs to be `let`, as `counterFour` is changed on line 45.

Reply:
updated